File: 프로그래머스/lv0/120841. 점의 위치 구하기/점의 위치 구하기.py
def solution(dot):
    answer = 0
    x, y = dot[0], dot[1]
    
    if x > 0 and y > 0:
        answer = 1
    elif x < 0 and y > 0:
        answer = 2
    elif x < 0 and y < 0:
        answer = 3
    elif x > 0 and y < 0:
        answer = 4
        
    return answer

File: 프로그래머스/lv0/120841. 점의 위치 구하기/점의 위치 구하기.swift
import Foundation

func solution(_ dot:[Int]) -> Int {
    var x: Int = dot[0]
    var y: Int = dot[1]
    var answer: Int = 0
    
    if x > 0 && y > 0 {
        answer = 1
    } else if x < 0 && y > 0 {
        answer = 2
    } else if x < 0 && y < 0 {
        answer = 3
    } else if x > 0 && y < 0 {
        answer = 4
    }
    
    return answer
}